### ISSUE TITLE
[Feature Request]: clear the conversation history

### ISSUE THREAD

**Issue by chagong** [microsoft/teams-ai#752]
When using teams-ai to build an AI chatbot, instead of append conversation history into openai request, people may want some control on it, like clear the chat history so that openai want process the previous history. Can teams-ai provide an interface to handle it easily?

**Comment by imadilkhalil** [comment]
Depends on how you want to achieve it. e.g. you can have

```
app.message('/forget', async (context: TurnContext, state: ApplicationTurnState) => {
    ConversationHistory.clear(state);
    await context.sendActivity(`forgot all conversation history.`);
});
```

When a user sends `/forget`, all conversation history will be erased.

**Comment by corinagum** [comment]
clear history is a feature that has been added and will be released for GA :)

**Comment by codedingt** [comment]
Hi, @corinagum, could you please confirm if this is available on GA 1.1.0?

Thanks!
